### DISCUSSION TITLE
question - is the rate limiter distributed?

### DISCUSSION THREAD

**Discussion by RichardWright** [OptimalBits/bull#2637] [Q&A]
Hi!

When using the [rate limiter](https://github.com/OptimalBits/bull/blob/13fe324a25373103fa7dcb4490e997db1ebfd3b8/docs/README.md?plain=1#L173) can this same rate limiter be accessed from seperate processes / boxes etc, so it could control in a distributed fashion, the http activity of a cluster? We currently use [bottleneck](https://www.npmjs.com/package/bottleneck) and are assessing bull as a replacement 

Thanks!

**Comment by manast** [comment]
It is distributed. But I recommend you to use BullMQ if you are new to Bull: https://docs.bullmq.io/guide/rate-limiting
